tests/cli/test_main_remove.py

```diff
@@ -106,3 +108,31 @@ def test_remove_globbed_package_names(
 def test_remove_nonexistent_env(conda_cli: CondaCLIFixture):
     with pytest.raises(EnvironmentLocationNotFound):
         conda_cli("remove", "--prefix=/non/existent/path", "--all")
+
+
+def test_remove_all_default_activation_env(
+    conda_cli: CondaCLIFixture,
+    tmp_env: TmpEnvFixture,
+    tmpdir: str,
+):
+    """Check that removing the default_activation_env raises an exception."""
+    with tmp_env() as env:
```

Comment:
The convention we've followed in the codebase is to call this variable `prefix`:

```suggestion
    with tmp_env() as prefix:
```